arbnode/delayed.go

```diff
@@ -8,6 +8,7 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	gomath "math"
```

Comment:
Here and in other files, why do you alias it to gomath, when there is not an ambiguous import anymore?

Reply:
Done.